docs/packaging/submitting-a-package.md

```diff
@@ -63,45 +94,78 @@ For files that must be removed, you must do so using git: `git rm someFile`
+
+Now that you've tested and reviewed your change, when you're happy with it, it's time to commit your changes with `git commit`.
+
+Make sure you provide a meaningful summary (with the package name) and a separate body to your commit message. Package update details should come from the upstream project release notes, and contain no links. There may also be a section for Solus specific work.
+
+There should be a summary line, a blank line, and then the rest of the commit message. Bullet point lists should start with a dash.
+
+Here is an example in our standard format:
 
-Once you're happy with your change, and you have verified locally that it works by having first built and
-installed it, it's time to commit your changes with `git commit`.
+```
+Update foo to 1.2.3
+
+**Changelog**
+
+Bugfixes:
+
+- Fixed a crash
+- Something else
+
+Enhancements:
+
+- Implemented a feature
+- Error when encountering a thing
+
+**Full release notes:**
+- [1.2.3](https://github.com/foo/foo/releases/tag/v1.2.3)
+
+## Test Plan
+
+- Launched the application
+- Exercised the UI
+- Exercised some feature
+```
 
-Make sure you provide a meaningful summary and a separate body to your commit message. For more information
-on suitable commit messages, please check the [tooling central documentation](https://github.com/solus-project/tooling-central/blob/master/README.rst#using-git).
+For more information on suitable commit messages, please check the [tooling central documentation](https://github.com/solus-project/tooling-central/blob/master/README.rst#using-git).
 
-- If you want to link this patch to an issue on the Dev Tracker, simply mention it in your commit message: `The inclusion of <somepackage> fixes #1234`
-- If you need a change to depend on another change, mention it in the commit message too: `Depends on #123`
+- If you want to link this pull request to an existing issue, simply mention it in your commit message (use the full URL): `The inclusion of <somepackage> fixes https://github.com/getsolus/packages/issues/123`
+- If you need a change to depend on another change, mention it in the commit message too (use the full URL): `Depends on https://github.com/getsolus/packages/issues/234`
 
-### Submitting for Review
+### Submitting the Pull Request for Review
 
-Now you have your changed committed, it's time to send it to us for review using by submitting a pull request to the GitHub repository.
+Now that you have your changed committed, it's time to send it to us for review by submitting a pull request to the GitHub repository.
 
-You will first need to fork the repository either from the GitHub web interface. Or by using the `gh` cli tool from the `github-cli` package.
+You will first need to fork the repository. You can do this either from the GitHub web interface, or by using the [`gh` cli tool](https://cli.github.com/manual/gh_repo_fork) from the `github-cli` package.
```

Comment:
This doesn't work the same for package updates and new packages. Additionally, it would simplify the process if we move this to the beginning, so: 1) fork, 2) clone your fork, 3) do updates, 4) commit, 5) push, 6) create PR. No fiddling with the remote.

Reply:
Agreed. I'm re-doing the create and update docs to re-order the steps and put forking first.